.babelrc

```diff
@@ -17,13 +20,36 @@
           "public": "./public"
         }
       }
-    ],
+    ]
   ],
   "env": {
     "development": {
       "plugins": [
         "react-hot-loader/babel"
       ]
+    },
+    "test": {
+      "presets": [
+        ["es2015"],
+        "react",
+        "stage-0"
+      ],
+      "plugins": [
```

Comment:
Re-think about plugins here.

Reply:
Let's talk about it a little bit later. 🤓 I think that migration to `babel-preset-env` will be in the next PR.